snap/snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: prometheus-bind-exporter
+version: git
```

Comment:
When I run `snapcraft`, which runs all the stages (pull, build, stage, prime), it fails with the following error:

```
Building prometheus-bind-exporter 
+ snapcraftctl build
+ go mod download
+ go install -p 8 -ldflags -linkmode=external ./...
Staging prometheus-bind-exporter 
+ snapcraftctl stage
Priming prometheus-bind-exporter 
+ snapcraftctl prime
Determining the version from the project repo (version: git).
fatal: not a git repository (or any of the parent directories): .git
Run the same command again with --debug to shell into the environment if you wish to introspect this failure.
```

When I changed the version to `0.4.git`, it worked. I think snaps should always be built against tagged versions, though. See [this example](https://github.com/canonical/juju-verify/blob/a06f21847c6d76eecb3dee171f80d4a0b828ce4f/snap/snapcraft.yaml#L36) on how to set a version via a build override.